subscription-descriptor-parser/src/main/java/uk/gov/justice/maven/generator/io/files/parser/SubscriptionDescriptorFileParserFactory.java

```diff
@@ -7,6 +7,14 @@
 
     @Override
     public SubscriptionDescriptorFileParser create() {
-        return new SubscriptionDescriptorFileParser();
+        return new SubscriptionDescriptorFileParser(subscriptionDescriptorFileValidator());
+    }
+
+    private YamlFileToJsonObjectConverter yamlFileToJsonObjectConverter() {
```

Comment:
is there any benefit of having this in a separate method?

Reply:
Inlined